### PR TITLE
Improve example text for "Image links"

### PR DESCRIPTION
**Pull request by cyberphor**
### Description

Updates [Image Links](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#image_links) section to make it more clear the example shown uses a locally hosted image. 

### Motivation

I am suggesting this change after trying the example provided myself. I quickly realized the file wasn't on my computer. Therefore, I felt the example was a little misleading. 

### Additional details

None. 

### Related issues and pull requests

None.

### CODE REVIEW

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks @cyberphor for adding the clarifying text!
I've added a few suggestions.

File: files/en-us/learn/html/introduction_to_html/creating_hyperlinks/index.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ This turns the heading into a link:
 
 ### Image links
 
-If you have an image you want to make into a link, use the {{htmlelement("a")}} element to wrap the image file referenced with the {{htmlelement("img")}} element.
+If you have an image you want to make into a link, use the {{htmlelement("a")}} element to wrap the local image file referenced with the {{htmlelement("img")}} element. The example below references an image file saved locally.
```
Comment (estelle) [inline files/en-us/learn/html/introduction_to_html/creating_hyperlinks/index.md:84]: I don't think this change is necessary. The first sentence requires no edits and should not include the word local. The `a` is used to create an element no matter where the image file is located.

If clarifying text is needed, in this case the  explanation could state the image is in the same directory as the HTML file. however, as best practices for file structure usually recommend storing images in an assets or images directory, maybe we should change the example to demonstrate these recommendations.

Comment (cyberphor) [reply files/en-us/learn/html/introduction_to_html/creating_hyperlinks/index.md:84]: Yes, after making the PR, I realized the word "local" was unnecessary. Yet, I still think some more context would be helpful (even if there was a sentence implying relative and absolute paths will be discussed later in the MDN material).

Comment (dipikabh) [reply files/en-us/learn/html/introduction_to_html/creating_hyperlinks/index.md:84]: I've made some suggestions here: https://github.com/mdn/content/pull/28580#discussion_r1296132975

**Review by dipikabh — APPROVED** [review]
Thanks @cyberphor!

Welcome to the MDN project 🎉

**Pull request merged**